src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorSingleStream.java

```diff
@@ -399,4 +332,127 @@ public Detector(
+                ssrcs = null;
+            }
+
+            // XXX The variable naming is chosen to keep the source code close to
+            // the original.
+            Detector estimator = it;
+
+            estimator.lastPacketTimeMs = nowMs;
+            this.incomingBitrate.update(packetLength, nowMs);
+
+            BandwidthUsage priorState = estimator.detector.getState();
+            long[] deltas = this.deltas;
+
+        /* long timestampDelta */ deltas[0] = 0;
+        /* long timeDelta */ deltas[1] = 0;
+        /* int sizeDelta */ deltas[2] = 0;
+
+            if (estimator.interArrival.computeDeltas(
+                    /**  "timestamp" is the rtp sendtime -
+                     * time the rtp packet is sent at the sender*/
+                    sendTime,
+                    /** "arrivalTime" is received time -
+                     * time recorderd by the receiver
+                     * when the packet is received */
+                    receivedTime,
+                    packetLength,
+                    deltas,
+                    nowMs))
+             {
+                double timestampDeltaMs
+                        = /* timestampDelta */ deltas[0] * kTimestampToMs;
+
+                estimator.estimator.update(
+                    /* timeDelta */ deltas[1],
+                        timestampDeltaMs,
+                    /* sizeDelta */ (int) deltas[2],
+                        estimator.detector.getState());
+                estimator.detector.detect(
+                        estimator.estimator.getOffset(),
+                        timestampDeltaMs,
+                        estimator.estimator.getNumOfDeltas(),
+                        nowMs);
+            }
+            if (estimator.detector.getState() == BandwidthUsage.kBwOverusing)
+            {
+                long incomingBitrateBps = this.incomingBitrate.getRate(nowMs);
+
+                if (priorState != BandwidthUsage.kBwOverusing
+                        || remoteRate.isTimeToReduceFurther(
+                        nowMs,
+                        incomingBitrateBps))
+                {
+                    // The first overuse should immediately trigger a new estimate.
+                    // We also have to update the estimate immediately if we are
+                    // overusing and the target bitrate is too high compared to what
+                    // we are receiving.
+                    updateEstimate(nowMs);
+                }
+            }
+        }  // synchronized (critSect)
+
+        return this.incomingBitrate.getRate(receivedTime);
+    }
+
+
+    /**
+     * {@Link rtcpTCCRemoteBitrateEstimator} estimates the remote bitrate
+     * of the receiver from received rtcp tcc feedback packet parameters.
+     * This acts for the send-side of the bridge
+     * @param ssrc_ is the SSRC of the source
+     * @param sendTime is the time the packet was sent by the bridge
+     * @param receivedTime is the time recorded by the receiver
```

Comment:
Same as above, make it clear that one is an RTP timestamp, the other is a local timestamp